README.md

```diff
@@ -143,6 +117,87 @@ In order to start up oneview-redfish-toolkit service, there is some mandatory co
  
   * **emailAddress**: Email address to contact the responsible for this server/certificate. This is an optional information. Will not be added to certificate if not informed. **Optional.**
 
+## Logging
+
+Logging configuration can be found in `logging.conf` file. The provided configuration enables INFO level at both console and file output, which will generate a `redfish.log` file at `$HOME/.config/oneview-redfish-toolkit/`.
+
+The application has two extended logs, where all logs will be created on the same directory `$HOME/.config/oneview-redfish-toolkit/`:
+
+#### Performance logger
+
+For each Redfish Toolkit API request the performance logger will log the elapsed time for each OneView SDK request triggered, the amount elapsed time for all OneView SDK requests, the overhead toolkit process elapsed time and the total elapsed time for the Redfish Toolkit API request. So we can monitoring the OneView performance and the redfish toolkit performance as well. It will be log on `redfish_performance.log` file once enabled on `logging.conf` by changing its level and the root log level as DEBUG. Log example:
+```
+2018-10-10 15:42:41,112 - perf - DEBUG   - Thread 139974930331392 - OneView request: server_profile_templates.get: 0.02929878234
+2018-10-10 15:42:42,304 - perf - DEBUG   - Thread 139974930331392 - OneView request: server_hardware.get_all: 0.03278422355
+2018-10-10 15:42:43,231 - perf - DEBUG   - Thread 139974930331392 - OneView request: server_hardware.get_all:  0.03367638587
+2018-10-10 15:42:43,291 - perf - DEBUG   - Thread 139974930331392 - OneView process: 0.09575939176
+2018-10-10 15:42:43,291 - perf - DEBUG   - Thread 139974930331392 - Redfish process: 0.01273488998413086
+2018-10-10 15:42:43,294 - perf - DEBUG   - Thread 139974930331392 - Total process: 0.10849428174
+```
+
+#### OneView Data logger
+
+The OneView data logger will log the result for each OneView SDK request triggered. So we can check the data retrievered for each Redfish API request. It will be log on `redfish_ov_data.log` file once enabled on `logging.conf` by changing its level and the root log level to DEBUG. Log example:
```

Comment:
Unfortunally, the same behavior that the performance log, the file is always created.

Reply:
Something like: "So we can check OneView data retrieved for each OneView SDK request."

Reply:
Done.